scripts/entrypoint.sh

```diff
@@ -39,6 +39,7 @@ Arguments:
     - "zeek": latest released Zeek version (default)
     - "zeek-lts": long-term support version
     - "zeek-nightly": latest nightly build
+    - "zeek-X.Y": explicitly versioned package
```

Comment:
For now Y=0. No strong opinion, but putting X.Y here is a bit over-promising. But might be possible with container images down the road.

Reply:
My impression (and hope) was that we would publish a package e.g., `zeek-6.1` as well (useful we publish patch versions for non-LTS releases as well which would otherwise be hard to consume). If that is not the case I can make this very explicit.

Reply:
Looks like that is not the case, so I changed the code.

Reply:
Leaving a bread-crumb: https://github.com/zeek/zeek-docs/pull/200#discussion_r1262199362

Think I'm still on the side of having feature releases be ephemeral is okay given how short lived they and due to the laxer stability guarantees.

Reply:
Interesting. Not having packages for non-LTS  releases does make it hard to configure CI using this action against a non-moving "latest release" though (`zeek` could jump from `zeek-6.2` to `zeek-7.0`). However, that's the situation we have 🤷.